pkg/bootstrap/config.go

```diff
@@ -201,11 +237,25 @@ func getStatsOptions(meta *model.NodeMetadata, nodeIPs []string) []option.Instan
 		return substituteValues(inclusionOption, "{pod_ip}", nodeIPs)
 	}
 
+	extraStatTags := make([]string, 0, len(DefaultStatTags))
+	extraStatTags = append(extraStatTags,
```

Comment:
does order matter?

Reply:
and are duplicates ok?

Reply:
Order does not matter. Duplicates don't matter unless we somehow produce duplicates and that would be a bug.

Reply:
what is a user puts a tag that is already in the default?

Reply:
That would be strange but harmless.